# Competition configuration options

options:
  insert_models_over_bins: true
  fill_demo_tray: false

orders:
  order_0:
    announcement_time: 0.0
    parts:
      part_0:
        type: part3
        pose:
          xyz: [0.1, -0.15, 0]
          rpy: [0, 0, 0]
      part_1:
        type: part4
        pose:
          xyz: [0.1, 0.15, 0]
          rpy: [0, 0, 0]

models_over_bins:
  bin7:
    models:
      piston_rod_part:
        xyz_start: [0.1, 0.1, 0.0]
        xyz_end: [0.5, 0.5, 0.0]
        rpy: [0, 0, 'pi/4']
        num_models_x: 3
        num_models_y: 4

time_limit: -1


# Competitor trial configuration options

arm:
  type: ur10
  initial_joint_states:
    linear_arm_actuator_joint: 1

sensors:
  break_beam:
    type: break_beam
    pose:
      xyz: [1.6, 2.25, 0.95]
      rpy: [0, 0, 'pi']
  proximity_sensor:
    type: proximity_sensor
    pose:
      xyz: [0.88, 2.32, 0.94]
      rpy: [0, 0, 0]
  logical_camera:
    type: logical_camera
    pose:
      xyz: [-0.3, 0.15, 1.93]
      rpy: [0, 'pi/2', 0]
  laser_profiler:
    type: laser_profiler
    pose:
      xyz: [1.21816, 4, 1.22]
      rpy: ['-pi', 'pi/2', 'pi/2']
